bundles/af-core/src/main/java/com/adobe/cq/forms/core/components/util/AbstractFormComponentImpl.java

```diff
@@ -116,7 +116,7 @@ protected void initBaseModel() {
         if (Boolean.TRUE.equals(unboundFormElement)) {
             dataRef = NULL_DATA_REF;
         }
-        getNameOrSetDefault();
+        getName();
```

Comment:
function name changes because getNameOrSetDefault was setting default value for others as well. Other test cases were failing